ra/ra.go

```diff
@@ -2016,10 +2015,7 @@ func (ra *RegistrationAuthorityImpl) RevokeCertByApplicant(ctx context.Context,
 		return nil, err
 	}
 
-	err = ra.purgeOCSPCache(ctx, cert, int64(issuerID))
-	if err != nil {
-		return nil, err
-	}
+	_ = ra.purgeOCSPCache(ctx, cert, int64(issuerID))
```

Comment:
We've resolved to stop rejecting new requests (at the purger) when the queue is full. But, what if the gRPC service itself is actually unavailable (e.g. networking, bug, bad config, etc.)? Would it be useful to emit the URLs that it failed to purge here so they can be purged manually?

Reply:
For reference: 1 cert is 3 URLs measuring <400 bytes, so 1-million would be <400 megabytes of text. If that would destroy our logging performance it's understandable.

Reply:
How about this: Let's file a ticket for the akamai-purger "never fail, because we'll drop the bottom of the stack instead" work, and add a TODO here referencing that ticket to change this to check the error at that time?

I've filed https://github.com/letsencrypt/boulder/issues/5979 for this purpose.